### PR TITLE
E 109

### PR DESCRIPTION
**Pull request by tdelabro**

### CODE REVIEW

File: frontend/src/components/PayoutTable/index.tsx [hunk 1]
```diff
@@ -113,17 +113,17 @@ const renderHeaders = (sorting: Sorting, applySorting: (field: Field) => void) =
 
 const renderPayments = (payments: Payment[], payoutInfoMissing: boolean) => {
   return payments.map(payment => (
-    <Line key={payment.id}>
-      <Cell> {dayjs.tz(payment.requestedAt, dayjs.tz.guess()).fromNow()} </Cell>
+    <Line key={payment.id} className="transition duration-200 hover:bg-white/4 outline-offset-0 hover:outline-2">
```
Comment (AnthonyBuisset) [inline frontend/src/components/PayoutTable/index.tsx:116]: maybe this could be a generic theme in the Line component

Comment (oscarwroche) [reply frontend/src/components/PayoutTable/index.tsx:116]: Yes, we are already applying a similar theme in Line when there is a link

File: frontend/src/components/ContributorsTable/index.tsx [hunk 2]
```diff
@@ -91,26 +92,28 @@ const renderContributors = (contributors: Contributor[]) => {
...
           </div>
-          {contributor.isRegistered && (
-            <div className="relative group/od-logo">
-              <img src={onlyDustLogo} className="h-3.5" />
-              <div className="invisible group-hover/od-logo:visible absolute top-5 -left-16 w-36 z-10">
-                <Tooltip>{T("contributor.table.userRegisteredTooltip")}</Tooltip>
-              </div>
+          <div className="flex space-x-1 items-end">
+            <div>
+              <span className="text-fuchsia-300">{contributor.login}</span>
```
Comment (AnthonyBuisset) [inline frontend/src/components/ContributorsTable/index.tsx:103]: fuschia is not on the design color palette. I think it is `spacePurple-200`

File: frontend/src/components/ContributorsTable/index.tsx [hunk 3]
```diff
@@ -91,26 +92,28 @@ const renderContributors = (contributors: Contributor[]) => {
...
+            {contributor.isRegistered && (
+              <div className="relative group/od-logo">
+                <img src={onlyDustLogo} className="h-3.5" />
+                <div className="invisible group-hover/od-logo:visible absolute top-5 -left-16 w-36 z-10">
+                  <Tooltip>{T("contributor.table.userRegisteredTooltip")}</Tooltip>
+                </div>
+              </div>
+            )}
+          </div>
+          <ExternalLinkLine className="text-fuchsia-700 invisible group-hover/line:visible" />
```
Comment (AnthonyBuisset) [inline frontend/src/components/ContributorsTable/index.tsx:114]: same here: `spacePurple-500`

File: frontend/src/components/Table/Line.tsx [hunk 4]
```diff
@@ -1,17 +1,20 @@
...
+  highlightOnHover?: number;
 }
 
-const Line: React.FC<Props> = ({ className, children, link }) => {
+const Line: React.FC<Props> = ({ className, children, highlightOnHover }) => {
   return (
     <tr
-      onClick={link ? linkClickHandlerFactory(link) : undefined}
-      className={`group/line border-b border-gray-800 ${link && "hover:bg-white/5 cursor-pointer"} ${className}`}
+      className={`group/line border-b border-gray-800
```
Comment (AnthonyBuisset) [inline frontend/src/components/Table/Line.tsx:12]: you can use classNames utility

**Review by AnthonyBuisset — APPROVED** [review]

File: frontend/src/components/ContributorsTable/index.tsx (hunk 2 above)
Comment (tdelabro) [reply frontend/src/components/ContributorsTable/index.tsx:103]: done

File: frontend/src/components/ContributorsTable/index.tsx (hunk 3 above)
Comment (tdelabro) [reply frontend/src/components/ContributorsTable/index.tsx:114]: done

File: frontend/src/components/Table/Line.tsx (hunk 4 above)
Comment (tdelabro) [reply frontend/src/components/Table/Line.tsx]: done

**Review by oscarwroche — APPROVED** [review]

**Pull request merged**